processing/fb_event_processor.coffee

```diff
@@ -54,9 +55,10 @@ exports.extractAndSendEvents = (feed) ->
             location: getLocationInfo FBevent
             isAllDay: FBevent.end_time?
             startTime: startTime
-            endTime: if FBevent.end_time then new Date(FBevent.end_time).getTime() else null
+            endTime: if FBevent.end_time then new Date(FBevent.end_time).getTime() else 0
```

Comment:
But if end_time is 0 for an ongoing event it will not be returned. Is that what you want?